tools/ansible/tasks/logging/test/logging_test.go

```diff
@@ -25,7 +28,22 @@ var requiredPluginList = map[string]string{
 	"fluent-plugin-prometheus":                 "",
 }
 
-// TestPluginsInstalled tests that the fluentd plugins were all installed correctly.
+// TestRubyInstalled tests if Ruby is installed correctly
+// This test should be run on a node after the logging playbook has been run against it
+func TestRubyInstalled(t *testing.T) {
+	// Pinning down version of Ruby Devkit to 2.5.7, 64 bit version
+	// Ruby installation path assumed to be default
+	var rubyExecutablePath = "C:\\Ruby25-x64\\bin\\ruby.exe"
```

Comment:
So, this test need to be changed every time we upgrade the ruby version in the ansible playbook?

Reply:
We are providing the path to download the Ruby Installer as a variable while running the playbook.
This test needs to be changed only when the way Ruby is installed changes for some specific version.